src/Microsoft.Framework.PackageManager/Program.cs

```diff
@@ -518,7 +519,7 @@ public int Main(string[] args)
                     command.InPlace = optInPlace.HasValue();
                     command.Framework = optFramework.Value();
 
-                    var success = command.ExecuteCommand();
+                    var success = command.ExecuteCommand(RuntimeEnvironmentHelper.IsMono(_runtimeEnv));
```

Comment:
Same here. Runtime info should be passed to ctor and ExecuteCommand() should take no parameter.

Please help do some cleanup BTW, let's rename this `WrapCommand.ExecuteCommand()` to `WrapCommand.Execute()` to remove redundancy in the name.